src/scripts/prefill.ts

```diff
@@ -0,0 +1,56 @@
+import 'dotenv/config';
+import { readFile } from 'fs/promises';
+import { getTextsBeforeDones } from '../helpers';
+import { insertToSupabase } from '../supabase';
+
+interface TgJsonExport {
+  name: string;
+  id: number;
+  messages: [
+    {
+      date_unixtime: string;
+      from?: string;
+      from_id?: string;
+      text: '' | ['' | { type: string; text: string }];
```

Comment:
```suggestion
      text: string | Array<string | { type: string; text: string }>;
```

It's not types, basically :)

`''` - is "empty string", while you need `string` I suppose. `['' | { type: string; text: string }]` <- and I'm not sure that this one is working. 🤔